axon-framework/axon-framework-commands/infrastructure.md

```diff
@@ -414,59 +376,242 @@ public DisruptorCommandBus commandBus(TransactionManager txManager, AxonConfigur
+As it is the default, configuring it is relatively straightforward:
 
+{% tabs %}
+{% tab title="Axon Configuration API" %}
+Declare dependencies:
 ```text
-axon.distributed.enabled=true
+<!--somewhere in the POM file-->
+<dependency>
+    <groupId>org.axonframework</groupId>
+    <artifactId>axon-server-connector</artifactId>
+    <version>${axon.version}</version>
+</dependency>
+<dependency>
+    <groupId>org.axonframework</groupId>
+    <artifactId>axon-configuration</artifactId>
+    <version>${axon.version}</version>
+</dependency>
+```
+
+Configure your application:
+```java
+// The AxonServerCommandBus is configured as Command Bus by default when constructing a DefaultConfigurer.
+Configurer configurer = DefaultConfigurer.defaultConfiguration();
 ```
+{% endtab %}
 
-There is one setting that is independent of the type of connector used:
+{% tab title="Spring Boot AutoConfiguration" %}
+By simply declaring the `axon-spring-boot-starter` dependency, Axon will automatically configure the `AxonServerCommandBus`:
 
 ```text
-axon.distributed.load-factor=100
+<!--somewhere in the POM file-->
+<dependency>
+    <groupId>org.axonframework</groupId>
+    <artifactId>axon-spring-boot-starter</artifactId>
+    <version>${axon.version}</version>
+</dependency>
 ```
 
-The load factor of a Distributed Command Bus is defaulted to `100`.
+{% endtab %}
+{% endtabs %}
 
-> **The Load Factor Explained**
+> **Disabling Axon Server**
+>
+> There are two options to disable Axon Framework's default of using the `AxonServerCommandBus`:
+>
+> 1. By excluding the `axon-server-connector` dependency.
+> 2. By setting `axon.server.enabled` to `false` when Spring Boot is used.
 >
-> The load factor defines the amount of load the instance would carry compared to others. For example, if you have a two machine set up, both with a load factor of 100, both will carry an equal amount of load. Increasing the load factor to 200 on both would still mean that both machines receive the same amount of load. Concluding, the load factor is intended to serve heterogeneous application landscapes with the means to distribute more load to faster machines than to slower machines.
+> When doing either of these, Axon will fallback to the not distributed [`SimpleCommandBus`](infrastructure.md#simplecommandbus), unless configured otherwise.
+
+#### Local Segment and Load Factor Configuration
+
+The [load factor](#load-factor) for the `AxonServerCommandBus` is defined through the `CommandLoadFactorProvider`.
+This interface allows us to define a different load factor per command message.
+This might proof useful if some commands should be targeted towards more often to one instance in favour of the other.
+
+The following should be done to configure a custom [local segment](#local-segment) and/or load factor:
+
+{% tabs %}
+{% tab title="Axon Configuration API" %}
+```java
+// Somewhere in the configuration class
+public CommandBus axonServerCommandBus(CommandBus localSegment,
```

Comment:
does it need to be a `@Bean`?

Reply:
Not in this case, as it is not the Spring Config section but the plain Axon Configuration API  :-)